sites/kit.svelte.dev/package.json

```diff
@@ -9,6 +9,7 @@
 	},
 	"devDependencies": {
 		"@sveltejs/adapter-auto": "workspace:*",
+		"@sveltejs/adapter-static": "workspace:^1.0.0-next.29",
```

Comment:
I think this should be `workspace:*` like others are

Reply:
This is simply rollback to previous commit.
https://github.com/sveltejs/kit/blame/cdc32a9f4f17d69be6ca1cae58bf3961155545a2/sites/kit.svelte.dev/package.json

But obviously `workspace:*` is better I think too.